zebra-chain/Cargo.toml

```diff
@@ -29,9 +29,11 @@ serde-big-array = "0.3.0"
 # ZF deps
 ed25519-zebra = "0.4"
 redjubjub = "0.1"
-equihash = { git = "https://github.com/ZcashFoundation/librustzcash.git", branch = "equihash-crate" }
+equihash = { git = "https://github.com/ZcashFoundation/librustzcash.git", branch = "equihash-errors" }
```

Comment:
This change will be reverted before this PR is merged but is needed for it to compile while https://github.com/ZcashFoundation/librustzcash/pull/1 is unmerged

Reply:
I merged ZcashFoundation/librustzcash#1.

Please revert this change, and then you can admin-merge, if you'd like :-)